stdpopsim/genomes.py

```diff
@@ -320,31 +356,34 @@ def species_contig(
             L_tot = 0
             r_tot = 0
             u_tot = 0
-            gc_tot = 0
+            gcf_tot = 0
             gcl_tot = 0
             for chrom_data in species.genome.chromosomes:
                 if chrom_data.id.lower() not in non_autosomal_lower:
                     L_tot += chrom_data.length
                     r_tot += chrom_data.length * chrom_data.recombination_rate
                     u_tot += chrom_data.length * chrom_data.mutation_rate
-                    if chrom_data.gene_conversion_rate is None:
-                        local_gc_rate = 0
-                    else:
-                        local_gc_rate = chrom_data.gene_conversion_rate
+                    if chrom_data.gene_conversion_fraction is not None:
+                        gcf_tot += (
+                            chrom_data.length * chrom_data.gene_conversion_fraction
+                        )
+                    if chrom_data.gene_conversion_length is not None:
```

Comment:
This might cause some confusing effects as we are using the mean of the gc fraction and not the actual gc rate.

Example:

Two chromosomes with the same length.
Chromosome A has a recombination rate of 10 and a gc_fraction of 0.1, i.e. a gc_rate of approx. 1.11
Chromosome B has a recombination rate of 1 and a gc_fraction of 0.5, i.e. a gc_rate of 1.
The generic contig will now have a recombination rate of 5.5 and a gc_fraction of 0.3 resulting in a gc_rate of approx. 2.35.

Reply:
The straightforward solution would be to calculate the actual gc_rates for each chromosome take the mean and calculate the corresponding gc_fraction from this.

Reply:
hah good call